huggingface_sb3/push_to_hub.py

```diff
@@ -7,7 +7,8 @@
 from pathlib import Path
 from typing import Any, Dict, Optional, Tuple, Union
 
-import gym
+import gym as gym26
```

Comment:
maybe a lazy import would be better? (so gym 0.26 is not required to use hf SB3)

Reply:
Good idea thanks, I wrote the lazy import (now gym as gym26 is only loaded if Key Error in entry_point() 😄